.gitignore

```diff
@@ -1,3 +1,11 @@
 .SD_Store
 xcuserdata
 .DS_Store
+
+### Xcode Patch ###
+*.xcodeproj/*
```

Comment:
This seems a bit strange - why is everything ignored here?

Reply:
Because when I've changed the scheme app language to russian, it changed the xcscheme file, and I think it doesn't should be tracked by git, Moreover the gitignore.io suggests those ignores

Reply:
@joeblau To my recollection you're working on gitignore, can you elaborate the rationale here? Ignoring everything inside `*.xcodeproj` means you also won't have a pbxproj file 🤔 

Makes sense to drop xcuserdata specifically but not sure about the rest.

Reply:
I think Shai is right here, you shouldn't be excluding the entire `*.xcodeproj/*` as this will ignore changes in the `*.pbxproj` file. I understand that Xcode changes the `xcshareddata` folder regarding the scheme changed, in your case the language but this is a different change so you should not have any problem working locally with this isn't?